full_snapshot.proto

```diff
@@ -479,6 +479,17 @@ message RelationStatistic {
 
   int32 frozenxid_age = 30; // The age of frozen xact ID for this relation (relative to latest stable xid)
   int32 minmxid_age = 31; // The age of minimum multixact ID for this relation (relative to latest stable mxid)
+
+  // For infrequently updated stats
+  int32 relpages = 40;                 // Size of the on-disk representation of this table in pages (of size BLCKSZ)
+  int64 reltuples = 41;                // Number of live rows in the table. -1 indicating that the row count is unknown
+  int32 relallvisible = 42;            // Number of pages that are marked all-visible in the table's visibility map
+  int32 relfrozenxid = 43;             // All transaction IDs before this one have been replaced with a permanent (“frozen”) transaction ID in this table
+  int32 relminmxid = 44;               // All multixact IDs before this one have been replaced by a transaction ID in this table
```

Comment:
Reviewing the `multixact.c` code (and specifically the wraparound handling in `GetNewMultiXactId`) it appears that MultiXact IDs do not have an epoch counter today, so I think we can keep this as 32-bit.

Reply:
Actually, from https://github.com/pganalyze/collector/pull/350#discussion_r1057403795, this should be either `uint32` or `int64` regardless, as it's `xid`. We use `uint32` for `minimum_multixact_xid` so this... should be `uint32` too if we want to match, but decided to go with `int64` as it matches more with what `RelationStatistic` message has